frontend/app/views/paypal/errorPage.scala.html

```diff
@@ -7,7 +8,7 @@
 	        <p>Sorry, there was a problem completing your PayPal payment. Please try again:</p>
 	        <a class="paypal-error-page__signup-link" href="@routes.Info.supporterRedirect(None)"><button class="action">Become a Supporter</button></a>
 	        <a class="paypal-error-page__signup-link" href="@routes.Joiner.enterPaidDetails(Partner())"><button class="action">Become a Partner</button></a>
-	        <a class="paypal-error-page__signup-link" href="@routes.Info.patron"><button class="action">Become a Patron</button></a>
+	        <a class="paypal-error-page__signup-link" href="@Links.patrons"><button class="action">Become a Patron</button></a>
```

Comment:
I think that linking to the new Patrons scheme from this page is confusing, because a user wouldn't land on this error page as part of the new Patrons flow.

We could remove Patrons links from this page altogether?

Reply:
I agree, once we've proved that there are absolutely no more possible ways to get to the patron page on this site. And given that some pages pull from the catalogue to provide links to these things, that might take a while 😖